### ISSUE TITLE
使用load上来的informer模型预测数据，每次结果均不同

### ISSUE THREAD

**Issue by H4Njx** [PaddlePaddle/PaddleTS#368]
##### 训练代码包括存储和预测
```python
from paddlets.models.forecasting.dl.informer import InformerModel as Informer
from paddlets import TSDataset
from paddlets.transform.sklearn_transforms import StandardScaler

import pandas as pd
from typing import List

def data_process(df:pd.DataFrame,scaler,time_col_name:str,target_col_list:List,observed_col_list:List,freq:str):
    data = TSDataset.load_from_dataframe(
        df,
        time_col=time_col_name,
        target_cols=target_col_list,
        observed_cov_cols=observed_col_list,
        freq=freq)

    train_data, test_data = data.split(0.8)
    scaler.fit(train_data)
    scaler_train_data = scaler.transform(train_data)
    scaler_test_data = scaler.transform(test_data)
    return scaler_train_data, scaler_test_data, scaler

if __name__ == "__main__":
    model = Informer(in_chunk_len=28,
                      out_chunk_len= 28,
                      eval_metrics=["mse","mae"],
                      max_epochs=3)
    df = pd.read_csv("/Informer/train/traindata.csv")
    scaler = StandardScaler()
    train_data, test_data ,scaler= data_process(df,scaler,"date",'y',None,"1d")
    model.fit(train_data,test_data)
    res = model.predict(test_data)
    inverse_transform_res = scaler.inverse_transform(res)
    model.save("informer.savedmodel")
    print(inverse_transform_res)
```
##### 预测结果为
```bash
                      y
2023-02-07  1425.993652
2023-02-08  1424.377930
2023-02-09  1421.871216
2023-02-10  1418.660522
2023-02-11  1418.290039
2023-02-12  1422.693726
2023-02-13  1424.804688
2023-02-14  1423.174194
2023-02-15  1422.863525
2023-02-16  1417.690918
2023-02-17  1416.067017
2023-02-18  1422.147583
2023-02-19  1424.315552
2023-02-20  1424.332520
2023-02-21  1419.405273
2023-02-22  1419.088623
2023-02-23  1413.171265
2023-02-24  1409.463013
2023-02-25  1409.834229
2023-02-26  1414.650269
2023-02-27  1409.096558
2023-02-28  1416.881958
2023-03-01  1416.226440
2023-03-02  1419.820190
2023-03-03  1417.339355
2023-03-04  1410.086182
2023-03-05  1419.711792
2023-03-06  1415.859253
```
##### 加载模型及预测代码为
```python
from paddlets.models.model_loader import load
import pandas as pd
from typing import List
from paddlets import TSDataset
from paddlets.transform.sklearn_transforms import StandardScaler

def data_process(df:pd.DataFrame,scaler,time_col_name:str,target_col_list:List,observed_col_list:List,freq:str):
    data = TSDataset.load_from_dataframe(
        df,
        time_col=time_col_name,
        target_cols=target_col_list,
        observed_cov_cols=observed_col_list,
        freq=freq)

    train_data, test_data = data.split(0.8)
    scaler.fit(train_data)
    scaler_train_data = scaler.transform(train_data)
    scaler_test_data = scaler.transform(test_data)
    return scaler_train_data, scaler_test_data, scaler

if __name__ == "__main__":

    df = pd.read_csv("/Informer/train/traindata.csv")

    scaler = StandardScaler()
    
    train_data, test_data ,scaler= data_process(df,scaler,"date",'y',None,"1d")
    trained_model = load("./informer.savedmodel")
    load_model_predres = trained_model.predict(test_data)
    print(test_data)
    inverse_transform_res = scaler.inverse_transform(load_model_predres)
    print(inverse_transform_res)
```
##### 预测结果
```bash
# 测试数据
                   y
date                
2022-09-30  1.259896
2022-10-01  1.547872
2022-10-02  1.455891
2022-10-03  1.478457
2022-10-04  1.543245
...              ...
2023-02-02  2.189494
2023-02-03  2.167281
2023-02-04  2.259263
2023-02-05  1.973050
2023-02-06  1.993500

[130 rows x 1 columns]
# 预测结果
                      y
2023-02-07  1421.905273
2023-02-08  1421.521362
2023-02-09  1423.071899
2023-02-10  1422.876709
2023-02-11  1415.927734
2023-02-12  1421.687500
2023-02-13  1419.604492
2023-02-14  1417.005981
2023-02-15  1419.268433
2023-02-16  1427.177124
2023-02-17  1423.654419
2023-02-18  1419.297363
2023-02-19  1421.166626
2023-02-20  1421.554077
2023-02-21  1416.492432
2023-02-22  1417.657349
2023-02-23  1414.936523
2023-02-24  1415.555908
2023-02-25  1418.315308
2023-02-26  1413.397705
2023-02-27  1411.110596
2023-02-28  1411.677124
2023-03-01  1417.716919
2023-03-02  1414.239746
2023-03-03  1417.350220
2023-03-04  1419.005615
2023-03-05  1416.818115
2023-03-06  1419.117310
```

**Comment by zy07604** [comment]
训练和预测之前需要设定随机种子，这样才能保持一致，可以增加以下两行代码：
`np.random.seed(2022)
paddle.seed(2022)`
